### ISSUE TITLE
Implement Math Algorithms

### ISSUE THREAD

**Issue by arnabuchiha** [sharadbhat/nodorithm#1]
- [x] GCD
- [x] LCM
- [x] Multiplication Expo Squaring
- [x] Mod Inverse 
@sharadbhat Can I work on this ?

**Comment by sharadbhat** [comment]
Yeah sure @arnabuchiha. Go ahead

**Comment by arnabuchiha** [comment]
I have completed it. Can you merge it and label as "hacktoberfest-accepted" ?

**Comment by sharadbhat** [comment]
Yes, I've noted it. I'm busy this week. I'll review it over the weekend.

P.S. this repository isn't opted in to the `hacktoberfest` topic. So, I don't think the label will count towards your goal.

**Comment by arnabuchiha** [comment]
Yeah sure. No the label "hacktoberfest-accepted" will help it.
